docs/articles/documentation/templates/boundtestrun.md

```diff
@@ -0,0 +1,9 @@
+### options.boundTestRun
+
+**Type**: Object
+
+You can use the `boundTestRun` option to call a client function from a Node.js callback.
+Assign the current [test controller](../reference/test-api/testcontroller/README.md)
+to the `boundTestRun` option to achieve this.
```

Comment:
Change to:
To use this option, assign the current [test controller](../reference/test-api/testcontroller/README.md)
to the `boundTestRun` option.